### PR TITLE
set binding tests to required

### PR DESCRIPTION
**Pull request by baude**
some small fix ups for binding tests and then make them required.

Signed-off-by: bbaude <bbaude@redhat.com>
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/bindings/test/pods_test.go [hunk 1]
```diff
@@ -57,6 +57,11 @@ var _ = Describe("Podman pods", func() {
 		podSummary, err := pods.List(bt.conn, nil)
 		Expect(err).To(BeNil())
 		Expect(len(podSummary)).To(Equal(1))
+
+		// Start the pod
+		_, err = pods.Start(bt.conn, newpod)
```
Comment (mheon) [inline pkg/bindings/test/pods_test.go:62]: Eek. This looks like a but in start for containers - I don't think this should be required

Comment (mheon) [reply pkg/bindings/test/pods_test.go:62]: A bug, even

Comment (mheon) [reply pkg/bindings/test/pods_test.go:62]: I'm fine merging for now, but we should definitely fix this.

File: pkg/bindings/test/manifests_test.go [hunk 2]
```diff
@@ -118,25 +118,27 @@ var _ = Describe("Podman containers ", func() {
...
-		digest := data.Manifests[0].Digest.String()
-		annoOpts := image.ManifestAnnotateOpts{OS: "foo"}
-		_, err = manifests.Annotate(bt.conn, id, digest, annoOpts)
-		Expect(err).To(BeNil())
-		list, err := manifests.Inspect(bt.conn, id)
-		Expect(err).To(BeNil())
-		Expect(len(list.Manifests)).To(BeNumerically("==", 1))
-		Expect(list.Manifests[0].Platform.OS).To(Equal("foo"))
-	})
+	// There is NO annotate endpoint, this could never work.:w
```
Comment (rhatdan) [inline pkg/bindings/test/manifests_test.go:121]: Why comment out, Why not just delete.

File: pkg/bindings/manifests/manifests.go [hunk 3]
```diff
@@ -125,23 +125,24 @@ func Push(ctx context.Context, name string, destination *string, all *bool) (str
...
-		return "", err
-	}
-	stringReader := strings.NewReader(optionsString)
-	response, err := conn.DoRequest(stringReader, http.MethodPost, "/manifests/%s/annotate", params, name)
-	if err != nil {
-		return "", err
-	}
-	return idr.ID, response.Process(&idr)
-}
+//func Annotate(ctx context.Context, name, digest string, options image.ManifestAnnotateOpts) (string, error) {
```
Comment (rhatdan) [inline pkg/bindings/manifests/manifests.go:130]: Why comment out, Why not just delete.

File: pkg/domain/infra/tunnel/manifest.go [hunk 4]
```diff
@@ -64,30 +63,32 @@ func (ir *ImageEngine) ManifestAdd(ctx context.Context, opts entities.ManifestAd
...
-		}
-		manifestAnnotateOpts.Annotation = annotations
-	}
-	updatedListID, err := manifests.Annotate(ctx, names[0], names[1], manifestAnnotateOpts)
-	if err != nil {
-		return updatedListID, errors.Wrapf(err, "error annotating %s of manifest list %s", names[1], names[0])
-	}
-	return fmt.Sprintf("%s :%s", updatedListID, names[1]), nil
+	return "", errors.New("not implemented")
+	//	manifestAnnotateOpts := image.ManifestAnnotateOpts{
```
Comment (rhatdan) [inline pkg/domain/infra/tunnel/manifest.go:70]: Why comment out, Why not just delete.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**